.github/workflows/hub-deploy-sandbox.yaml

```diff
@@ -0,0 +1,42 @@
+name: Deploy-sandbox
+
+on:
+  workflow_dispatch:
+
+defaults:
+  run:
+    working-directory: sourcecode/hub
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 8.2
+          tools: composer:v2
+          coverage: none
+      
+      - name: Require Vapor CLI
+        run: composer global require laravel/vapor-cli
+
+      - name: Install Project Dependencies
+        run: composer install --no-interaction --prefer-dist --no-dev --optimize-autoloader
```

Comment:
Will these steps have any effect? This just installs dependencies and stuff in the CI environment. Doesn't `vapor deploy` ensure the same steps in `vapor.yml` are run in the image that ends up being deployed?

Reply:
That sounds like a valid assumption, but the [Vapor documentation](https://docs.vapor.build/1.0/projects/deployments.html#example-with-github-actions) keeps these in their GH example, and in the other CI examples.

Reply:
I think vapor-cli interacts with Laravel in some ways, so this is probably why it's done that way. The event caching and npm build can probably be removed from the workflow, then.

Reply:
As in, remove the whole line `npm ci && npm run build && rm -rf node_modules`, or just the `npm build` part?

Reply:
The whole line